lists/na/na-namra.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": {
+    "en": "Namibia Revenue Agency ",
+    "local": "English"
```

Comment:
This field is for the name of the Registry in the domestic language. So if another name for the Namibia Revenue Agency is not being supplied, please remove this field.